tests/unit/parser.js

```diff
@@ -3673,22 +3674,22 @@ exports["for of as esnext"] = function (test) {
     "for (const x = 1, y = 2 of [1,2,3,4]) print(x + y);"
   ];
   TestRun(test)
-    .addError(6, "Invalid for-of loop left-hand-side: initializer is forbidden.")
-    .addError(7, "Invalid for-of loop left-hand-side: more than one ForBinding.")
-    .addError(8, "Invalid for-of loop left-hand-side: initializer is forbidden.")
+    .addError(7, "Invalid for-of loop left-hand-side: initializer is forbidden.")
```

Comment:
@jugglinmike not sure what you mean? Sorry! ;- I changed this test to avoid it outputting new errors that are unrelated to for-of, because the test cases all re-used the same variable names.

Are you suggesting I split the test up into multiple tests? I could, but its alot of grunt work..